enterprise/cmd/executor/vm-image/install.sh

```diff
@@ -195,6 +200,86 @@ function preheat_kernel_image() {
   docker pull "weaveworks/ignite:${IGNITE_VERSION}"
 }
 
+## Configures the CNI explicitly and adds the isolation plugin to the chain.
+## This is to prevent cross-network communication (which currently doesn't happen
+## as we only have 1 bridge).
+function configure_cni() {
+  mkdir -p /etc/cni/net.d
+  cat <<EOF >/etc/cni/net.d/10-ignite.conflist
+{
+  "cniVersion": "0.4.0",
+  "name": "ignite-cni-bridge",
+  "plugins": [
+    {
+      "type": "bridge",
+      "bridge": "ignite0",
+      "isGateway": true,
+      "isDefaultGateway": true,
+      "promiscMode": true,
+      "ipMasq": true,
+      "ipam": {
+        "type": "host-local",
+        "subnet": "10.61.0.0/16"
+      }
+    },
+    {
+      "type": "portmap",
+      "capabilities": {
+        "portMappings": true
+      }
+    },
+    {
+      "type": "firewall"
```

Comment:
As per https://github.com/sourcegraph/sourcegraph/pull/39857/files#r939002098, I think we could just do the following:
```suggestion
      "type": "firewall",
      "backend": "iptables",
      "ingressPolicy": "same-bridge"
```
taken from https://github.com/containernetworking/plugins/pull/584/files#diff-f9500a054cb234fd4e59147ce80f26d5ed4eba3bcb033b7f3edb27ebc9fd61a2R82 which is linked to from cni-isolation README

Reply:
solved on slack, currently ignite doesn't support a recent enough version of this. We will come back to this eventually.